README.md

```diff
@@ -1,12 +1,12 @@
- [![npm version](https://badge.fury.io/js/wonka-ui.svg)](https://badge.fury.io/js/wonka-ui)
+git  [![npm version](https://badge.fury.io/js/wonka-ui.svg)](https://badge.fury.io/js/wonka-ui)
```

Comment:
I guess this "git" was not supposed to be here. Same in l.9.

Reply:
Fixed